### ISSUE TITLE
Should the ecosystem move to an organization?

### ISSUE THREAD

**Issue by asinghvi17** [rofinn/FilePathsBase.jl#178]
Hey, it may make sense to move the FilePaths ecosystem to an org (maybe JuliaIO, or spin up a new JuliaFileAccess org or something) for maintainability and to reduce bus factor.

Just a thought, it's not particularly urgent.  Happy to facilitate if needed.

**Comment by rofinn** [comment]
Possibly. I feel like JuliaIO could make sense, though I don't believe I'm in that org currently. I haven't personally found adding packages to orgs really helps address the maintenance issue, it's still a tragedy of the commons problem. That being said, it can be a convenient way of lowering the bus factor, rather than adding individual folks to specific repos.

**Comment by asinghvi17** [comment]
Yeah, it is quite a problem - I guess the downside of having widely used packages is that there are a lot more bugs for comparatively fewer developers per user...

In case you'd like to move the repos to JuliaIO, let me know and I can get the ball rolling over there!
